### PR TITLE
Jmprieur/update readme md

### PR DESCRIPTION
**Pull request by jmprieur**
- Updating the instructions to use the new portal.
- Updating the instructions to use MSAL.NET 3.x

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,22 +47,46 @@ From your shell or command line:
 ### [OPTIONAL] Step 2:  Register the sample on the app registration portal
 
 You can run the sample as is with its current settings, or you can optionally register it as a new application under your own developer account.
-Create a new app at [apps.dev.microsoft.com](https://apps.dev.microsoft.com), or follow these [detailed steps](https://azure.microsoft.com/en-us/documentation/articles/active-directory-v2-app-registration/).  Make sure to:
 
-- Copy down the **Application Id** assigned to your app, you'll need it in the next optional step.
-- Add the **Native Application** platform for your app.
+#### Choose the Azure AD tenant where you want to create your applications
```
Comment (danieldobalian) [inline README.md]: Will MSA people know this is an AAD tenant?

Comment (jmprieur) [reply README.md]: That's a good question. what would you recommend?

File: README.md [hunk 2]
```diff
@@ -47,22 +47,46 @@ From your shell or command line:
...
+
+As a first step you'll need to:
+
+1. Sign in to the [Azure portal](https://portal.azure.com) using either a work or school account or a personal Microsoft account.
+1. If your account is present in more than one Azure AD tenant, select `Directory + Subscription` at the top right corner in the menu on top of the page, and switch your portal session to the desired Azure AD tenant.
+1. In the left-hand navigation pane, select the **Azure Active Directory** service, and then select **App registrations (Preview)**.
+
+#### Register the client app (active-directory-xamarin-native-v2)
+
+1. In **App registrations (Preview)** page, select **New registration**.
```
Comment (danieldobalian) [inline README.md]: Drop the `(preview)` I think as per guidance. Also, I think you're supposed to link to the new blade.

File: README.md (hunk 1 above)
Comment (danieldobalian) [reply README.md]: "Choose the directory for your apps"

File: README.md [hunk 3]
```diff
@@ -47,22 +47,46 @@ From your shell or command line:
...
+1. When the **Register an application page** appears, enter your application's registration information:
+   - In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `active-directory-xamarin-native-v2`.
+   - In the **Supported account types** section, select **Accounts in any organizational directory and personal Microsoft accounts (e.g. Skype, Xbox, Outlook.com)**.
+    > Note that if there are more than one redirect URIs, you'd need to add them from the **Authentication** tab later after the app has been created succesfully.
```
Comment (jennyf19) [inline README.md]: >s [](start = 55, length = 1)

drop the s

Comment (jennyf19) [inline README.md]: and a s -> successfully

File: README.md (hunk 1 above)
Comment (jmprieur) [reply README.md]: @danieldobalian : I get your point, but if I don't know anything about identity, I might think that directory means folder? or not? (I would have thought so when I joint)

File: README.md (hunk 3 above)
Comment (jmprieur) [reply README.md]: fixed

Comment (jmprieur) [reply README.md]: thanks fixed

**Pull request merged**